themes/shared/forms.scss

```diff
@@ -28,8 +28,10 @@
   border-top-color: #727272;
   border-bottom-color: #dadada;
   background: white url("/img/themes/fresh/textbox-bg.png") repeat-x top;
+  background-image: linear-gradient(to bottom, #b4b4b4 0px, #fff 2px);
```

Comment:
My question is: why do we have `/img/themes/fresh/...` in `shared` CSS? It's either shared thing, then image should be in shared img folder, or this bg is just for fresh theme, then CSS should be in fresh folder.